packages/wallets/src/index.scss

```diff
@@ -1 +1,13 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@import '@deriv/quill-design/dist/quill-design.css';
+@import '@deriv/quill-design/quill-tailwind/styles.css';
+@import '@deriv/quill-design/quill-tailwind/fonts.css';
+
 @import './styles/devices.scss';
+svg {
```

Comment:
let'e don't merge quill icons and components in one PR.

Reply:
this is not about quill icons, there has been a conflict between deriv-app and quill css reset, this fixes the conflict.

Reply:
There was some issues when applying the base styles, `@tailwind base` and it is affecting svgs. We resolved it by adding this css rule to resolve the issue however this is a temporary solution. Ideally there is no need to add css.